lib/spree/data_visualisations/data_visualisation.rb

```diff
@@ -2,6 +2,26 @@ module Spree
   module DataVisualisations
     class DataVisualisation
 
+      # Permet d'obtenir les dates de l'année financière
+      def get_fiscal_year(reference)
+        today = Date.today
+        
+        from = Date.new(Date.today.year, 6, 30)
+        to = from + 1.day
+
+        if today < from
+          from = from - 1.year
+          to = to - 1.year
+        end
+
+        if reference == 'from'
```

Comment:
je trouve ca weird comme fonctionnement, on peu laisser de même, mais j'aurais plus retourner un hash {from: from, to: to}, comme t'aurais pu l'utiliser direct dans les filters.